usage/launcher/src/test/java/org/apache/brooklyn/launcher/BrooklynWebServerTest.java

```diff
@@ -146,11 +146,13 @@ public void verifyHttpsFromConfig() throws Exception {
     @Test
     public void verifyHttpsCiphers() throws Exception {
         brooklynProperties.put(BrooklynWebConfig.HTTPS_REQUIRED, true);
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_PROTOCOLS, "XXX");
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_CIPHERS, "XXX");
```

Comment:
The point of the test is to check whether the server actually uses the configured ciphers, protocols. So non-existing ciphers are configured which should cause the handshake to fail.
The exception is coming from the client, connecting to jetty. It should be independent of the jetty version as this is at the ssl connection level which is handled by the JVM.
On my system the root cause is `Caused by: java.io.EOFException: SSL peer shut down incorrectly` because the server can't match the ciphers offered by the client and immediately closes the connection.
So keep the properties and if needed adjust the test to assert the correct exception.